shopinvader_payment/components/payment_transaction_event_listerner.py

```diff
@@ -28,10 +28,10 @@ def _confirm_and_invalidate_session(self, sale_order):
             # once the cache is reset.
             collection = _PseudoCollection("shopinvader.backend", self.env)
             work = self.work.work_on(collection=collection)
-            setattr(work, "shopinvader_backend", shopinvader_backend)
+            work.shopinvader_backend = shopinvader_backend
             # another ugly hack: the partner here should be taken
             # from current client request
-            setattr(work, "partner", sale_order.partner_id)
+            work.partner = sale_order.partner_id
```

Comment:
Flake8 called this B010. @simahawk was there any particular reason to call `setattr` ?

Reply:
nope, Indeed I fixed it recently here https://github.com/shopinvader/odoo-shopinvader-payment/pull/39

Reply:
BTW in any case this piece of code should die soon as pointed out again here https://github.com/shopinvader/odoo-shopinvader-payment/pull/41